test/Sentry.AspNetCore.Tests/Sentry.AspNetCore.Tests.csproj

```diff
@@ -1,12 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <!-- Running these tests on Mono just hangs -->
-  <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
-    <TargetFrameworks>net461;$(TargetFrameworks)</TargetFrameworks>
-  </PropertyGroup>
-
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net5.0;netcoreapp3.1;netcoreapp2.1</TargetFrameworks>
+    <TargetFrameworks>net6.0;net5.0;netcoreapp3.1;netcoreapp2.1;net461</TargetFrameworks>
   </PropertyGroup>
```

Comment:
Note that `Condition="'$(OS)' == 'Windows_NT'"` is unsupported.  The correct way is `Condition="$([MSBuild]::IsOSPlatform('windows'))"`.  In this case though, we can just include it directly.  The comment about the tests hanging on Mono appears to be not the case anymore.